data/en/test_average_equal_or_less_validation_against_total.json

```diff
@@ -0,0 +1,93 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "survey_id": "0",
+    "title": "Grouped validation against total test survey",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "A survey that tests the average of breakdown question answers against a total",
+    "sections": [{
+        "id": "default-section",
+        "groups": [{
+            "id": "group",
+            "title": "Validate average against total",
+            "blocks": [{
+                    "type": "Question",
+                    "title": "Target Total",
+                    "id": "total-block",
+                    "description": "",
+                    "questions": [{
+                        "id": "total-question",
+                        "title": "Total",
+                        "type": "General",
+                        "answers": [{
+                            "id": "total-answer",
+                            "label": "Total",
+                            "mandatory": true,
+                            "type": "Number",
+                            "decimal_places": 2,
+                            "max_value": {
+                                "value": 100
+                            }
+                        }]
+                    }]
+                },
+                {
+                    "type": "Question",
+                    "title": "Calculated Total",
+                    "id": "breakdown-block",
+                    "questions": [{
+                        "id": "breakdown-question",
+                        "title": "Breakdown",
+                        "type": "Calculated",
+                        "calculated": {
+                            "calculated_type": "average",
```

Comment:
Could make this more specific and call it "aggregate_function". It's more specific but the terminology is less accessible to a general audience. Not sure that matters though. @dwyeradam you might have thoughts on this?

Reply:
In this instance I actually think readability is more important. As we have less and less involvement in creating the schema then understanding of the terms in them will be a greater issue.